cli/examples/device-screenResolution-sequence1.json

```diff
@@ -1,6 +1,6 @@
 {
   "methods": {
-    "device.screenResolution": {
+    "device.onScreenResolutionChanged": {
```

Comment:
These files are meant to say, "Mock Firebolt", please first return this value, then this value, then this value... when someone calls this method. In this case, the goal was for the device.screenResolution call to return these values in order, not to fire events in a sequence. The syntax for a sequence of events is different.